packaging/click/osmscout-server.conf

```diff
@@ -0,0 +1,8 @@
+start on started unity8
+respawn
+env LD_LIBRARY_PATH=/opt/click.ubuntu.com/osmscout-server.jonnius/current/lib/@ARCH_TRIPLET@:$LD_LIBRARY_PATH
+
+script
+	chdir /opt/click.ubuntu.com/osmscout-server.jonnius/current/
+	exec ./usr/bin/osmscout-server
```

Comment:
this should be

```
exec ./usr/bin/osmscout-server --listen
```

Otherwise you will be running full server with RAM consumption possibly hitting 1GB or so. listen will just wait for connections and fork full process on request. on idle, full process will close and handle over to small socket waiting process

Reply:
Added